SUMMARY.md

```diff
@@ -186,9 +218,23 @@ Biscuit {
 }
 ```
 
+Let's unpack what's displayed here:
+
+ - `symbols` carries a list of symbols used in the biscuit.
+ - `authority` carries information provided by the token creator. It gives the initial scope of the bicuit.
+ - `blocks` carries a list of blocks, which can refine the scope of the macaroon.
+
+Here, `authority` provides the initial block, which can be refined in subsequent blocks.
+
 A block comes with new symbols it adds to the system (there's a default symbol
 table that already contains values like `#authority` or `#operation`). It can
-contain facts, rules and caveats.
+contain facts, rules and caveats. A block contains:
+
+ - `symbols` a block can introduce new symbols (TODO are those symbols usable in following blocks?)
```

Comment:
symbols added in a block are sable in the following blocks, and the library makes sure a symbol is not declared twice when deserializing

Reply:
in this case, isn't the `symbols` part of the biscuit equivalent to the symbols part of the `authority` block?

Reply:
the symbols for the whole token also include the symbol table used by the token creator and the verifier. They need to use the same one, and it contains some often used values. Here the authority symbols list only contains `"user_id"`, the rest comes from the symbol table